.eslintrc.js

```diff
@@ -42,5 +42,6 @@ module.exports = {
   globals: {
     process: 'readonly',
     Buffer: true,
+    global: true,
```

Comment:
Let's enable global only in test files. Could you update the config?

Reply:
okay